Assets/Mirror/Components/NetworkProximityChecker.cs

```diff
@@ -26,6 +26,9 @@ public enum CheckMethod
 
         [Tooltip("Enable to force this object to be hidden from players.")]
         public bool forceHidden;
+        
+        [Tooltip("Always keep local player on his own observer list.")]
+        public bool alwaysKeepLocalPlayerObserver = true;
```

Comment:
Please dont set the default value to true, omit it and let it be false instead. This is to preserve the current behaviour for existing users, since this field didnt exist before it wont have a serialized false value for them.